apps/spruce/src/constants/externalResources.ts

```diff
@@ -158,7 +159,7 @@ export const getHoneycombSystemMetricsUrl = (
   )}&omitMissingValues`;
 };
 
-export const adminSettingsURL = `${getEvergreenUrl()}/admin`;
+export const adminSettingsURL = `${getSpruceURL()}/admin-settings`;
```

Comment:
keep this one as-is since we still need to be able to access the old Admin Settings page for the duration of the project
```suggestion
export const adminSettingsURL = `${getEvergreenUrl()}/admin`;
```

Reply:
oop good idea thank you